lib/utils/avatar.vue

```diff
@@ -1,42 +1,51 @@
 <style lang="sass">
-  @import '../../styleguide/colors';
-
-  .avatar {
-    border-radius: 50%;
-    flex: 0 0 40px;
-    height: 40px;
-    width: 40px;
-
-    &.avatar-small {
-      flex: 0 0 24px;
-      height: 24px;
-      width: 24px;
-    }
-
-    &.avatar-default {
-      align-items: center;
-      background-color: $md-grey-500;
-      color: $pure-white;
-      display: flex;
-      justify-content: center;
-    }
+  .kiln-avatar {
+    flex: 0 0 auto;
   }
 </style>
 
 <template>
-  <img v-if="url" class="avatar" :class="'avatar-' + size" :src="url" />
-  <div v-else class="avatar avatar-default" :class="'avatar-' + size">
-    <ui-icon icon="person"></ui-icon>
-  </div>
+  <avatar class="kiln-avatar" :username="username" :src="imageURL" :size="pixelSize" :rounded="true"></avatar>
 </template>
 
 <script>
-  import UiIcon from 'keen/UiIcon';
+  import _ from 'lodash';
```

Comment:
don't think we need to load all of lodash

Reply:
kiln intelligently compiles only the parts of lodash we use

https://github.com/clay/clay-kiln/blob/master/webpack.config.js#L15

https://github.com/clay/clay-kiln/blob/master/.babelrc#L16

Reply:
Yeah, the combination of packages is actually super cool and means we don't have to worry about which bits of lodash we're using